src/muse/readers/csv.py

```diff
@@ -869,7 +870,7 @@ def check_utilization_and_minimum_service_factors(data, filename):
     if "utilization_factor" not in data.columns:
         raise ValueError(
             f"""A technology needs to have a utilization factor defined for every
-             timeslice. Please check file {filename}."""
+             timeslice. Please check files: {filename}."""
```

Comment:
A few of the other functions further down need a similar change

Reply:
sorted